src/i18n/index.ts

```diff
@@ -21,7 +21,7 @@ const i18n = createI18n({
   // ...
 });
 export const setLocale = (language: Language): void => {
-  i18n.global.locale.value = language;
+  i18n.global.locale = language;
```

Comment:
not working when change to english/french. downgrade vue-i18n to 9.1.7 and change back to value style will solve problem.